axelor-account/src/main/resources/views/AccountingReport.xml

```diff
@@ -81,43 +76,37 @@
         onChange="action-move-Live-report-record-set-date-to"
-        requiredIf="[12,15,19,20,21,22,23,24,25].includes(reportType.typeSelect)"/>
+        requiredIf="[12,15,19,20,21,22,23,24].includes(reportType.typeSelect)"/>
       <field name="dateTo"
         readonlyIf="(statusSelect == 2 &amp;&amp; [18,19].includes(reportType.typeSelect)) || (reportType.typeSelect == 3000 &amp;&amp; reportType.comparison != 4)"
-        requiredIf="[12,15,19,20,21,22,23,24,25].includes(reportType.typeSelect)"/>
+        requiredIf="[12,15,19,20,21,22,23,24].includes(reportType.typeSelect)"/>
       <field name="isComparativeBalance" widget="inline-checkbox" colSpan="12"
         showIf="reportType.typeSelect == 2"/>
-
+      <panel name="otherPeriodPanel" title="Comparative period"
+        showIf="(reportType.typeSelect == 2 &amp;&amp; isComparativeBalance ) || (reportType.typeSelect == 3000 &amp;&amp; !reportType.useLegacyCustomReports)"
+        colSpan="12" canCollapse="true">
+        <field name="otherYear" colSpan="6"
+          onChange="action-accounting-report-record-other-period-dates"
+          onSelect="action-accounting-report-attrs-other-year-domain" form-view="year-account-form"
+          grid-view="year-account-grid"/>
+        <field name="otherPeriod" colSpan="6" canView="false"
+          onChange="action-accounting-report-record-other-period-on-change"
+          onSelect="action-accounting-report-attrs-other-period-domain" form-view="period-form"
+          grid-view="period-grid"/>
+        <field name="otherDateFrom" colSpan="6"/>
+        <field name="otherDateTo" colSpan="6"/>
+      </panel>
     </panel>
-
     <panel-tabs name="mainPanelTab">
       <panel name="filterPanel" title="Filter">
-        <panel name="otherPeriodPanel" title="Comparative period" colSpan="12"
-          canCollapse="true" collapseIf="true"
-          showIf="(reportType.typeSelect == 2 &amp;&amp; isComparativeBalance ) || (reportType.typeSelect == 3000 &amp;&amp; !reportType.useLegacyCustomReports)">
-          <field name="otherYear"
-            onChange="action-accounting-report-record-other-period-dates"
-            onSelect="action-accounting-report-attrs-other-year-domain"
-            grid-view="year-account-grid" form-view="year-account-form" colSpan="6"/>
-          <field name="otherPeriod"
-            onChange="action-accounting-report-record-other-period-on-change"
-            onSelect="action-accounting-report-attrs-other-period-domain" form-view="period-form"
-            grid-view="period-grid" canView="false" colSpan="6"/>
-          <field name="otherDateFrom" colSpan="6"/>
-          <field name="otherDateTo" colSpan="6"/>
-        </panel>
-
-        <panel name="accountingPanel" title="Accounting" colSpan="12" canCollapse="true"
-          hideIf="reportType.typeSelect == 25">
+        <panel name="accountingPanel" title="Accounting" colSpan="12" canCollapse="true">
```

Comment:
please add attribute : `hideIf="reportType.typeSelect == 25"` to panel accountingPanel